packages/mobile/ios/Podfile

```diff
@@ -51,6 +51,9 @@ target 'AudiusReactNative' do
     )
     __apply_Xcode_12_5_M1_post_install_workaround(installer)
     # flipper_post_install(installer)
+    installer.pods_project.build_configurations.each do |config|
```

Comment:
interesting, so i think this make sense, but do we want to apply this wholesale to every build config? in the case of LD linking failing, a way around it is to manually install the pod, basically. we had to do this for one of the pods above

Reply:
looking at pbxcodeproj and seems like we actually have these defined: `				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;` i wonder if maybe this is a mistake? and in which case your fix might actually be the patch for it! thoughts?